pdh_check/datadog_checks/pdh_check/pdh_check.py

```diff
@@ -21,4 +21,4 @@ def __init__(self, name, init_config, agentConfig, instances=None):
                 (counterset, None, inst_name, dd_name, mtype) for inst_name, dd_name, mtype in instance['metrics']
             )
 
-        PDHBaseCheck.__init__(self, name, init_config, agentConfig, instances, counter_list)
+        super(PDHCheck, self).__init__(name, init_config, instances, counter_list=counter_list)
```

Comment:
adding key to counter list to go around a bug in the base class that's being fixed here https://github.com/DataDog/integrations-core/pull/6162